doc/keepeye.txt

```diff
@@ -44,10 +44,14 @@ To customize the KeepEye message (only if `g:keepeye_callback` is not set ):
 >
   let g:keepeye_message = 'SAVE YOUR EYES, TAKE A BREAK'
 <
-To play a bell sound (only if [mpv](https://mpv.io/) is installed, and only if `g:keepeye_callback` is not set)
+To play a bell sound (default player is [mpv](https://mpv.io/). It will play only if `g:keepeye_callback` is not set)
```

Comment:
List of players

Reply:
I guess we don't need a list of players anymore since we let the user choose which player is better for him/her \o/

Reply:
You right, it was for the previous PR, before we discuss about it